src/main/java/se/sundsvall/comfactfacade/api/model/Signatory.java

```diff
@@ -0,0 +1,51 @@
+package se.sundsvall.comfactfacade.api.model;
+
+import java.util.List;
+import java.util.Objects;
+
+import jakarta.validation.constraints.NotEmpty;
+
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder(setterPrefix = "with")
+@Schema(description = "A Signatory related to the signing process.")
+public class Signatory extends Party {
+
+
+	@NotEmpty
+	@ArraySchema(schema = @Schema(implementation = Identification.class, description = "The means of identification to use to identify the signatories."))
```

Comment:
I may be wrong, but is the description necessary here, won't it be taken from the `@Schema` annotation on the `Identification`-class?

Reply:
You are indeed correct